packages/syft/setup.cfg

```diff
@@ -68,6 +67,7 @@ syft =
     PyYAML==6.0.1
     azure-storage-blob==12.19.1
     ipywidgets==8.1.2
+    rich==13.7.1
```

Comment:
Added `rich` here as it is used in `warnings.py`. Earlier we were getting it from hagrid but that dependency is getting removed above. cc @shubham3121